channel/state/outcome/outcome_test.go

```diff
@@ -163,18 +163,53 @@ func TestExitDecode(t *testing.T) {
 	}
 }
 
+var a = Allocations{{ // [{Alice: 2, Bob: 3}]
+	Destination:    types.Destination(common.HexToHash("0x0a")),
+	Amount:         big.NewInt(2),
+	AllocationType: 0,
+	Metadata:       make(types.Bytes, 0)}, {
+	Destination:    types.Destination(common.HexToHash("0x0b")),
+	Amount:         big.NewInt(3),
+	AllocationType: 0,
+	Metadata:       make(types.Bytes, 0)}}
+
 func TestTotal(t *testing.T) {
-	a := Allocations{{ // [{Alice: 2, Bob: 3}]
-		Destination:    types.Destination(common.HexToHash("0x0a")),
-		Amount:         big.NewInt(2),
-		AllocationType: 0,
-		Metadata:       make(types.Bytes, 0)}, {
-		Destination:    types.Destination(common.HexToHash("0x0b")),
-		Amount:         big.NewInt(3),
-		AllocationType: 0,
-		Metadata:       make(types.Bytes, 0)}}
+
 	total := a.Total()
 	if total.Cmp(big.NewInt(5)) != 0 {
 		t.Errorf(`Expected total to be 5, got %v`, total)
 	}
 }
+
+func TestAffordsFor(t *testing.T) {
+
+	allocationsWithRepeatedEntry := append(a, a[0]) // [{Alice: 2, Bob: 3, Alice: 2}]
```

Comment:
Given above comment, could also use test case `[{Alice:2, Bob:3, Alice: 4}]` to communicate intent around these distinct cases.